hypervisor/include/arch/x86/ioapic.h

```diff
@@ -33,6 +59,15 @@ void gsi_unmask_irq(uint32_t irq);
 extern uint8_t pic_ioapic_pin_map[NR_LEGACY_PIN];
 
 #ifdef HV_DEBUG
+/**
+ * @brief Get information of ioapic
+ *
+ * It's for debug only. 
```

Comment:
Extra space to be removed from the end of the line
```suggestion
 * It's for debug only.
```